config/rabbitmq.php

```diff
@@ -93,4 +93,15 @@
         'passphrase' => env('RABBITMQ_SSL_PASSPHRASE', null),
     ],
 
+    'receive' => [
```

Comment:
Probably better place for this configuration is `options.connection` or `options.worker`?

Reply:
I prefer options.connection, as it has timeout setting.